src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -202,14 +202,15 @@ private function getOrderCustomer(Order $order): OrderCustomerForViewing
 
         $customerStats = $customer->getStats();
 
+        $totalSpentSinceRegistration = Tools::convertPrice($customerStats['total_orders'], $order->id_currency);
         return new OrderCustomerForViewing(
             $customer->id,
             $customer->firstname,
             $customer->lastname,
             $genderName,
             $customer->email,
             new DateTimeImmutable($customer->date_add),
-            $this->locale->formatPrice(Tools::convertPrice($customerStats['total_orders'], $order->id_currency), $currency->iso_code),
+            !is_null($totalSpentSinceRegistration) ? $this->locale->formatPrice($totalSpentSinceRegistration, $currency->iso_code) : null,
```

Comment:
ALways prefer `null` comparaison instead of `is_null` function
```suggestion
            $totalSpentSinceRegistration !== null ? $this->locale->formatPrice($totalSpentSinceRegistration, $currency->iso_code) : null,
```

Reply:
Fixed